protocols/v2/noise-sv2/src/responder.rs

```diff
@@ -331,7 +367,13 @@ impl Responder {
     // certificate validity period, and a cryptographic signature. The signature is created using
     // the responder's static public key and authority keypair, ensuring that the responder's
     // identity and certificate validity are cryptographically verifiable.
-    fn get_signature(&self, version: u16, valid_from: u32, not_valid_after: u32) -> [u8; 74] {
+    fn get_signature<R: rand::Rng + rand::CryptoRng>(
```

Comment:
making this method #[inline] here, will help to reduce the stack space.

Reply:
done